calcom/cal.com
# Fixes login callbackUrl and redirects to app root not website

## What does this PR do?
- Added back callbackUrl only local, change WEBSITE_URL -> APP_URL root.

## Type of change

## How should this be tested?
Try to login adding a custom callback url, shouldn't work.

You should be redirected to app.cal not to call after login
